### PR TITLE
LPS-116420 Build lang

### PR DESCRIPTION
**Pull request by jkappler**

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:relevant

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js [hunk 1]
```diff
@@ -86,17 +92,37 @@ function PageTypeSelector(props) {
...
 						)}
-						<ClayDropDown.Item
-							data-value={Liferay.Language.get('configure')}
-							key={Liferay.Language.get('configure')}
-							onClick={handleOnConfigureClick}
-							title={Liferay.Language.get('configure')}
-						>
-							{Liferay.Language.get('configure')}
-						</ClayDropDown.Item>
+						{props.addCollectionLayoutURL !== '' && (
```
Comment (victorg1991) [inline modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js]: If we want to cover undefined and null cases maybe using coercion makes it simpler:
`props.addCollectionLayoutURL`

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js [hunk 2]
```diff
@@ -86,17 +92,37 @@ function PageTypeSelector(props) {
...
+							</ClayDropDown.Item>
+						)}
 					</ClayDropDown.ItemList>
 				</ClayDropDown>
 			</div>
+			<div className="autofit-col ml-2">
+				<ClayLink
+					borderless
+					displayType="unstyled"
+					href="javascript:;"
```
Comment (victorg1991) [inline modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js:122]: If it is a link why not just setting the url here?
`href={props.addCollectionLayoutURL}`

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/main.scss [hunk 3]
```diff
@@ -88,6 +96,10 @@ $product-menu-pages-administration-link-height: 72px;
 		margin-top: 20px;
 		padding-top: 16px;
 
+		a {
```
Comment (victorg1991) [inline modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/main.scss:99]: don't you have to use .configure-link here? The class is added but not used anywhere

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js (hunk 1 above)
Comment (jkappler) [reply modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js]: 👍

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js (hunk 2 above)
Comment (jkappler) [reply modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js]: 👍

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/main.scss (hunk 3 above)
Comment (jkappler) [reply modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/main.scss:99]: 👍

**Pull request closed without merging**